src/react/components/pages/editorPage/editorPage.test.tsx

```diff
@@ -807,6 +818,73 @@ describe("Editor Page Component", () => {
             }));
         });
     });
+
+    describe("Active Learning", () => {
+        let wrapper: ReactWrapper = null;
+        let editorPage: ReactWrapper<IEditorPageProps, IEditorPageState> = null;
+
+        const objectDetectionMock = ObjectDetection as jest.Mock<ObjectDetection>;
+        Object.defineProperty(objectDetectionMock.prototype, "loaded", {
+            get: () => {
+              return true;
+            },
+        });
+        objectDetectionMock.prototype.detect = jest.fn(() => {
+            return [{bbox: [0, 0, 1, 1], class: "label", score: 1}];
+        });
+
+        const assetTestCache = new Map<string, IAsset>();
+        beforeEach(async () => {
+            const testProject = MockFactory.createTestProject("TestProject");
+            testProject.activeLearningSettings.predictTag = true;
+            const store = createStore(testProject, true);
+            const props = MockFactory.editorPageProps(testProject.id);
+            wrapper = createComponent(store, props);
+
+            editorPage = wrapper.find(EditorPage).childAt(0);
+            await waitForSelectedAsset(wrapper);
+            wrapper.update();
+
+            assetTestCache.clear();
+
+            document.createElement = jest.fn((elementType) => {
```

Comment:
Consider moving this whole block to the `MockFactory` and reusing it in the different places